torch_geometric/nn/conv/lightgcn_conv.py

```diff
@@ -0,0 +1,63 @@
+from torch_geometric.typing import Adj
+
+import torch
+from torch import Tensor
+from torch_geometric.nn import MessagePassing
+
+
+class LightGCNConv(MessagePassing):
```

Comment:
I prefer `LGConv` (Light Graph Convolution):
```suggestion
class LGConv(MessagePassing):
```